storm-client/test/jvm/org/apache/storm/executor/ExecutorTransferMultiThreadingTest.java

```diff
@@ -126,14 +126,15 @@ public void testExecutorTransfer() throws InterruptedException {
         }
 
         //give producers enough time to insert messages into the queue
-        executorService.awaitTermination(100, TimeUnit.MILLISECONDS);
+        executorService.awaitTermination(1000, TimeUnit.MILLISECONDS);
```

Comment:
Observation: that is really a long time for 10 messages. Wonder why it is that slow.

Reply:
This test intentionally submits 10 threads with a 5 thread pool to create race condition between threads. 100ms was just fine in my laptop or VM. On travis, I guess the resource is much limited so the thread contention is more observable. 1s maximum should be good enough. We should be able to lower it but since this value is just a maximum, 1s should be harmless.

Reply:
As you said, this is harmless for testing. Hopefully the core performance has not been significantly affected.

Reply:
Yeah, we haven't changed anything related to core performance. This is a new unit test. It works well in my laptop but fails sometimes on travis. That is mostly an environment difference.